### ISSUE TITLE
there is something wrong with the code

### ISSUE THREAD

**Issue by Jarvis-Cai** [mmkamani7/ResNet-Cifar#1]
I'm sorry that this code can't run.Because there is a strange package named resnet_model.what is that?

**Comment by peterzhang2029** [comment]
I guess you can delete the line directly.  However I met the problom that the  final result of eval dataset can just reach accuracy = 0.6114.

**Comment by mmkamani7** [comment]
That is correct @peterzhang2029. There are some discrepancies with Tensorflow Keras modules in some versions that I will update the code soon to resolve them.
